app/models/organization.rb

```diff
@@ -3,4 +3,8 @@ class Organization < ActiveRecord::Base
   validates :department_name, presence: true
   validates :team_name, presence: true
   validates :agency_name, uniqueness: { scope: [:department_name, :team_name] }
+
+  def to_s
+    "#{agency_name} | #{department_name} | #{team_name}"
```

Comment:
Conclusion from Slack: there is no consistency in this because government. We should combine the fields. Will do that separately.